src/ContextualPopup/ContextualPopup.js

```diff
@@ -568,10 +606,10 @@ module.exports = kind(
 		// show a transparent scrim for modal popups if
 		// {@link module:moonstone/ContextualPopup~ContextualPopup#scrimWhenModal} is `true`, else show a
 		// regular scrim.
-		if (this.modal && this.scrimWhenModal) {
+		if (this.modal && this.scrimWhenModal && !this.scrim) {
```

Comment:
I think we could potentially have two scrims appear if we changed the value of `this.scrim` and then proceeded to change the value of `spotlightModal`, for example.